set.mm

```diff
@@ -40343,6 +40350,14 @@ choice between (what we call) a "definitional form" where the shorter
       BCDJPQCOABEKLACDMBCDMN $.
   $}
 
+  ${
+    rabbii.1 $e |- ( ph <-> ps ) $.
+    $( Equivalent wff's correspond to equal restricted class abstractions.
+       (Contributed by Peter Mazsa, 1-Nov-2019.) $)
+    rabbii $p |- { x e. A | ph } = { x e. A | ps } $=
+      ( wb cv wcel a1i rabbiia ) ABCDABFCGDHEIJ $.
+  $}
+
```

Comment:
This is only one step longer than `~rabiia`, is it sufficient to promote it?

Reply:
I ran the minimize script for it, and it found 71 proofs which could be minimized (at least the number of bytes). Therefore, I think we can have it in main set.mm. I often looked for it an did not find it, so I was happy to find it in PM's mathbox now.

Reply:
I see! Then of course it makes sense.